pkg/pool-stable/contracts/StableMath.sol

```diff
@@ -447,4 +447,29 @@ library StableMath {
 
         _revert(Errors.STABLE_GET_BALANCE_DIDNT_CONVERGE);
     }
+
+    function _calcAllTokensInGivenExactBptOut(
```

Comment:
Why not have `_computeProportionalAmountsIn`?

Reply:
I'm using the same name as WeightedMath. Change that one too?

Reply:
Yes!